src/main/java/LinkedList/LinkedList.java

```diff
@@ -37,19 +37,62 @@ public boolean includes(int value) {
     }
 
     // this method will print the entire contents of the linked list
-    public ArrayList<Integer> print(){
+    public void print(){
 
         //find head
         Node current = this.head;
 
-        ArrayList<Integer> printArr = new ArrayList<>();
-
         //output all values of the list
         while (current != null) {
-            printArr.add(current.data);
+            System.out.print(current.data + " ");
             current = current.next;
         }
-        System.out.println(printArr.toString());
-        return printArr;
+        System.out.println();
+    }
+
+    // this method takes in a value and inserts it at the end of the list
+    public void append(int value) {
+
+        Node current = this.head;
+
+        while (current != null) {
```

Comment:
tiny optimization: could go `while (current.next != null)` and move the conditional logic outside of the loop